devel/addon/kyverno/policy.yaml

```diff
@@ -0,0 +1,725 @@
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+  name: require-default-proc-mount
+  namespace: cert-manager
+spec:
+  background: true
+  rules:
+  - match:
+      resources:
+        kinds:
+        - Pod
+    name: check-proc-mount
+    validate:
+      message: Changing the proc mount from the default is not allowed. The fields
+        spec.containers[*].securityContext.procMount and spec.initContainers[*].securityContext.procMount
+        must not be changed  from `Default`.
+      pattern:
+        spec:
+          =(initContainers):
+          - =(securityContext):
+              =(procMount): Default
+          containers:
+          - =(securityContext):
+              =(procMount): Default
+  validationFailureAction: enforce
+---
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  annotations:
+    policies.kyverno.io/category: Pod Security Standards (Restricted)
+    policies.kyverno.io/description: Containers should be forbidden from running with
+      a root primary or supplementary GID.
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+  name: require-non-root-groups
+  namespace: cert-manager
+spec:
+  background: true
+  rules:
+  - match:
+      resources:
+        kinds:
+        - Pod
+    name: check-runasgroup
+    validate:
+      message: "Running with root group IDs is disallowed. The fields\t spec.securityContext.runAsGroup,
+        spec.containers[*].securityContext.runAsGroup,\t and spec.initContainers[*].securityContext.runAsGroup
+        must be empty\t or greater than zero."
+      pattern:
+        spec:
+          =(initContainers):
+          - =(securityContext):
+              =(runAsGroup): '>0'
+          =(securityContext):
+            =(runAsGroup): '>0'
+          containers:
+          - =(securityContext):
+              =(runAsGroup): '>0'
+  - match:
+      resources:
+        kinds:
+        - Pod
+    name: check-supplementalGroups
+    validate:
+      message: "Adding of supplemental group IDs is not allowed. The field\t spec.securityContext.supplementalGroups
+        must not be defined."
+      pattern:
+        spec:
+          =(securityContext):
+            =(supplementalGroups):
+            - "null"
+  - match:
+      resources:
+        kinds:
+        - Pod
+    name: check-fsGroup
+    validate:
+      message: Changing to root group ID is disallowed. The field spec.securityContext.fsGroup
+        must be empty or greater than zero.
+      pattern:
+        spec:
+          =(securityContext):
+            =(fsGroup): '>0'
+  validationFailureAction: enforce
+---
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  annotations:
+    policies.kyverno.io/category: Pod Security Standards (Restricted)
+    policies.kyverno.io/description: Containers must be required to run as non-root
+      users.
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+  name: require-run-as-non-root
```

Comment:
👍🏼 so this policy would have been triggered if we didn't have `securityContext.runAsNonRoot`